src/frontend/services/wishlist-services/getWishlistService.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+export const getWishlistService = async (token) => {
+  try {
+    const { data } = await axios.get("/api/user/wishlist", {
+      headers: {
+        authorization: token,
+      },
+    });
+    return data.wishlist;
+  } catch (error) {
+    console.error(error.message);
```

Comment:
We should remove console.log before commit. We can store error in state & show it on view.